stingray/crossspectrum.py

```diff
@@ -1,19 +1,23 @@
-from __future__ import division
+from __future__ import division, absolute_import, print_function
 
 import numpy as np
 import scipy
 import scipy.stats
 import scipy.fftpack
 import scipy.optimize
 
-import stingray.lightcurve as lightcurve
-import stingray.utils as utils
-from stingray.exceptions import StingrayError
-from stingray.gti import cross_two_gtis, bin_intervals_from_gtis, check_gtis
+from .lightcurve import Lightcurve
+from .utils import rebin_data, simon
+from .exceptions import StingrayError
+from .gti import cross_two_gtis, bin_intervals_from_gtis, check_gtis
 
```

Comment:
To be honest, I think we should leave the absolute imports rather than relative imports here. This is also recommended by PEP-8. Are there good reasons for _not_ doing absolute imports?